backend/src/database/state-migrations/11-add-map-image-is-locked.ts

```diff
@@ -0,0 +1,22 @@
+import type { Action, UUID } from 'digital-fuesim-manv-shared';
+import type { Migration } from './migrations';
+
+export const addMapImageIsLocked11: Migration = {
+    actions: (_initialState, actions) => {
+        actions.forEach((action) => {
+            if ((action as Action | null)?.type === '[MapImage] Add MapImage') {
+                Object.assign((action as { mapImage: object }).mapImage, {
+                    isLocked: false,
+                });
```

Comment:
Dust

I'm personally still of the opinion that typings in the migrations aren't useful.

The `Object.assign()` syntax is also a bit strange but does make it easier to not use `any` typings.

-> Personal preferences.